sdk/react/panels/videoView.js

```diff
@@ -125,6 +125,20 @@ class VideoView extends React.Component {
     }
   };
 
+  onSeekPressed = (isForward) => {
+    const seekValue = (isForward) ? this.props.config.skipControls.skipForwardTime :  this.props.config.skipControls.skipBackwardTime * (-1);
+    const currentPlayhead = this.props.playhead;
+    let resultedPlayhead = currentPlayhead + seekValue;
+    if (resultedPlayhead < 0) {
+      resultedPlayhead = 0;
+    } else if (resultedPlayhead > this.props.duration) {
+      resultedPlayhead = this.props.duration;
+    }
+    const resultedPlayheadPercent = resultedPlayhead / this.props.duration;
```

Comment:
Is the duration always > 0? Is there a way to call this in between videos? I'm worried about an edge case of dividing by 0.

Reply:
@pilievOoyala You a right. I'm adding check

Reply:
@pilievOoyala added check duration on 0 value.